README.md

```diff
@@ -20,9 +19,18 @@ This is a Gatsby.js web development. In order to run this code you have to insta
    `npm install -g gatsby-cli`
 1. Download this repository   
 1. Have a text editor installed, i.e. [VSCode](https://code.visualstudio.com/)
```

Comment:
Maybe you can add 
Clone this repository
git clone https://github.com/air4children/site
Also, the recommended text editor VSCode but alternatively one can use favorite text editor (e.g , atom, pycharm, etc).

Reply:
Yeah, thanks for this siggestion. I added [here](https://github.com/air4children/site/pull/5/commits/c988c4901982b6a22b7797df5424fcafa03b9a70) more information.